examples/cmd/main.go

```diff
@@ -233,13 +234,13 @@ type bodyFunc = func(logger log.Logger, downloadersPool *DownloadersPool) error
 
 func (c *DownloaderCmd) ExecuteBody(logger log.Logger, fn bodyFunc) error {
 	ctx := context.Background()
-	var db *sql.DB
+	var storer github.Storer
 	if c.DB == "" {
 		log.Infof("using stdout to save the data")
-		db = nil
+		storer = &store.Stdout{}
 	} else {
 		var err error
```

Comment:
I think we no longer need this line, do we?

Reply:
And if I'm not wrong, we could also Close `db` in `defer`, no matter if we got an error or not (e.g. If I understood it properly, we're not closing it if `database.Migrate(c.DB)` succeeded)

Reply:
yes, thanks

Reply:
fixed